iati-activities-schema.xsd

```diff
@@ -1799,7 +1799,9 @@
                       <xsd:documentation xml:lang="en">
                         The baseline value.
 
-                        This should be a numeric value if the measure is quantitative.
+                        The @value must be omitted for qualitative measures
+                        The @value must be included for non-qualitative measures
+                        The @value should be a valid number for all non-qualitative measures
```

Comment:
So I suspect on the reference site, this will render as something like this:

> The baseline value.
>
> The @value must be omitted for qualitative measures The @value must be included for non-qualitative measures The @value should be a valid number for all non-qualitative measures

Which looks a bit odd. 2x carriage returns (and some full stops) would look nicer.

More generally, these are enforceable rules, so this could be added to the IATI-Rulesets repo, and these would then appear in the Rules section of the docs instead.

Reply:
@andylolz are they 'if' statements. e.g. If your 'thing' is qualitative then......? How would you write a machine readable for them when only of three options is true?

Reply:
It looks like the second of the three rules is included in the standard ruleset already. See: IATI/IATI-Rulesets#55.

I suspect it doesn’t get output to the reference site because it requires some updates to the script that converts the rules to English ([this one](https://github.com/IATI/IATI-Rulesets/blob/version-2.03/iatirulesets/text.py)) and these updates weren’t made.

The first rule wasn’t addressed, see: IATI/IATI-Rulesets#52 Although the justification for not addressing it is very flimsy – it would have been trivial to fix this.

The third is a “should”, so that’s fine.